core/src/main/scala/com/avast/cloud/metrics/datadog/api/Timer.scala

```diff
@@ -0,0 +1,9 @@
+package com.avast.cloud.metrics.datadog.api
+
+import scala.language.higherKinds
+
+trait Timer[F[_]] {
+  def time[A](f: F[A], tags: Tag*): F[A]
+
+  def recordExecutionTime(timeInMs: Long, tags: Tag*): F[Unit]
```

Comment:
why not nanos?

Reply:
also, does it measure  failure as well?

Reply:
it doesn't, this is very bare method that just records elapsed time 

ms because datadog doesn't have nano precision

Reply:
But that doesn't mean we can't take nanos as argument.